test/functional/specs/Personalization/C205529.js

```diff
@@ -31,9 +31,11 @@ test("Test C205529: Receive offer based on device", async () => {
   const alloy = createAlloyProxy();
   await alloy.configure(config);
   const result = await alloy.sendEvent({
-    xdm: {
```

Comment:
I have added a PR in Konductor that filters out device details from XDM, since those are captured in other place within Target Delivery API. Please check this doc: http://developers.adobetarget.com/api/delivery-api/#operation/execute. In the Delivery API we already capture screen width, height, orientation etc, there is no need to duplicate this data as Delivery API parameters. I changed the code so we don't have to adjust Target activity. As an alternative we could still use XDM, but adjust Target activity to use other keys instead of `device.screenWidth`.